src/cli/bunx_command.zig

```diff
@@ -148,43 +149,87 @@ pub const BunxCommand = struct {
 
     pub fn exec(ctx_: bun.CLI.Command.Context, argv: [][*:0]const u8) !void {
         var ctx = ctx_;
+
+        const cli = try CommandLineArguments.parse(ctx.allocator);
+
         var requests_buf = bun.PackageManager.UpdateRequest.Array.init(0) catch unreachable;
-        var run_in_bun = ctx.debug.run_in_bun;
+        var run_in_bun = ctx.debug.run_in_bun or cli.run_in_bun;
 
         var passthrough_list = try std.ArrayList(string).initCapacity(ctx.allocator, argv.len);
+
+        var bin_name = [1]string{""};
```

Comment:
Is this okay? Seems weird but the strategy was already in use for `package_name_for_update_request`.